pkg/orchestration/wiring/wiringutil/knownshapes/bindable_iam_accessible.go

```diff
@@ -10,13 +13,34 @@ const (
 // +k8s:deepcopy-gen=true
 // +k8s:deepcopy-gen:interfaces=github.com/atlassian/voyager/pkg/orchestration/wiring/wiringplugin.Shape
 type BindableIamAccessible struct {
-	wiringplugin.ShapeMeta           `json:",inline"`
+	wiringplugin.ShapeMeta `json:",inline"`
+	Data                   BindableIamAccessibleData `json:"data"`
+}
+
+// +k8s:deepcopy-gen=true
+type BindableIamAccessibleData struct {
 	wiringplugin.BindableShapeStruct `json:",inline"`
 	//IAMRoleARN    BindingProtoReference
 	//IAMProfileARN BindingProtoReference
 	IAMPolicySnippet wiringplugin.BindingProtoReference
 }
 
+func NewBindableIamAccessible(resourceName smith_v1.ResourceName, IAMPolicySnippetPath string) *BindableIamAccessible {
+	return &BindableIamAccessible{
+		ShapeMeta: wiringplugin.ShapeMeta{
+			ShapeName: BindableIamAccessibleShape,
+		},
+		Data: BindableIamAccessibleData{
+			BindableShapeStruct: wiringplugin.BindableShapeStruct{
+				ServiceInstanceName: wiringplugin.ProtoReference{
+					Resource: resourceName,
+					Path:     "metadata.name",
```

Comment:
Set `Example` please.

Reply:
Set it to what? I can't find any example of using this field in our codebase.